library/common/extensions/filters/http/test_event_tracker/config.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <string>
+
+#include "source/extensions/filters/http/common/factory_base.h"
+
+#include "library/common/extensions/filters/http/test_event_tracker/filter.h"
+#include "library/common/extensions/filters/http/test_event_tracker/filter.pb.h"
+#include "library/common/extensions/filters/http/test_event_tracker/filter.pb.validate.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace TestEventTracker {
+
+/**
+ * Config registration for the EventTracker filter. @see NamedHttpFilterConfigFactory.
+ */
+class TestEventTrackerFilterFactory
+    : public Common::FactoryBase<
+          envoymobile::extensions::filters::http::test_event_tracker::TestEventTracker> {
+public:
+  TestEventTrackerFilterFactory() : FactoryBase("test_event_tracker") {}
+
+private:
+  ::Envoy::Http::FilterFactoryCb createFilterFactoryFromProtoTyped(
+      const envoymobile::extensions::filters::http::test_event_tracker::TestEventTracker& config,
+      const std::string& stats_prefix, Server::Configuration::FactoryContext& context) override;
+};
+
+DECLARE_FACTORY(TestEventTrackerFilterFactory);
+
+} // namespace TestEventTracker
+} // namespace HttpFilters
+} // namespace Extensions
+} // namespace Envoy
```

Comment:
what are these?

Reply:
These comments explain what namespace a given `}` refers to. A few lines higher you should see how all of these namespaces are opened using `namespace Envoy` etc. It's a common practice to have these namespace related comments in Envoy Mobile repo I think.